tests/ui/specs/Popup/Popup-specs.js

```diff
@@ -1015,8 +1036,10 @@ describe('Popup', function () {
 				Page.waitForOpen(popup);
 
 				expectNoneScrimOpen(popupCommon);
+				Page.waitForFocused(popup.buttonOK);
 
-				Page.spotlightDown(); // OK button must have focus after Popup opens. This should be fixed and this line replace with `Page.spotlightRight();`.
+				Page.spotlightRight();
+				Page.waitForFocused(popup.buttonCancel);
 				Page.spotlightDown();
 
 				expect(popup.buttonCancel.isFocused()).to.be.true();
```

Comment:
Here also, we need to change the expectation and description.
```suggestion
				Page.waitForFocused(popupCommon.buttonPopup8);

				expect(popupCommon.buttonPopup8.isFocused()).to.be.true();
```